### PR TITLE
Reorganized documentation layout of Visualizing Results

### LINKED ISSUE #14896: We should break up the monolithic "Visualizing results" doc page
We've talked about this for ages, but this page, https://www.metabase.com/docs/latest/users-guide/05-visualizing-results.html, has gotten massively long now that we have 16 distinct visualizations(!).

We should add a new section to the [Users Guide](https://www.metabase.com/docs/latest/users-guide/start.html) with the heading "Visualizing results" and create a separate page for each visualization. I suspect that'll make it easier for folks to find what they're looking for, either when searching or browsing, and will make it simpler to link to articles on specific visualizations.

This will also give us the opportunity to elaborate on documentation for individual chart types, and should fix https://github.com/metabase/metabase/issues/9582 in the process.

### PR DESCRIPTION
**Pull request by d-zh21**
Changes
- added a visualizing results header and each visualization as a bullet point on the users guide page
- created different files for each individual visualization
- deleted the previous "Visualizing Results" page
- deleted the visualizing results bullet on the users guide page

Closes #14896

- - -
This change is [Reviewable](https://reviewable.io/reviews/metabase/metabase/22971)

### CODE REVIEW

**Comment by d-zh21** [comment]
@mazameli, @salsakran 

Hi, this is my first pull request on github.
Would either of you be the right person to review this?
I only made documentation changes so I'm not sure why the tests are failing.
I appreciate any help. Thanks

**Comment by mazameli** [comment]
Thanks for the PR, @d-zh21! The right person to review this is likely @jeff-bruemmer, so I've requested his review.

**Review by jeff-bruemmer — CHANGES_REQUESTED** [review]
Thanks for pitching in, @d-zh21. In order to get this PR merged, it still needs a fair amount of work.

Here are the changes I'm requesting:

- Reinstate the original 05-visualizing results page. This will prevent a lot of links from breaking. The page also included some info that applies to all visualizations, and it serves as a landing page where we can list links to each viz page.
- Create a directory `users-guide/visualizations` to store these pages to keep the repo organized.
- Remove the `v#` prefixes. E.g., `users-guide/v05-table.md` => `users-guide/visualizations/table.md`. The docs were originally numbered because they were sequenced, but we're trying to move away from that (brittle) structure.
- After you relocate the files, you'll need to update the image links in each file to insure they work. Bonus, you could create an image directory to house all of the viz page images.
- Update the links from the start page to the new files. 
- Include a link to the landing page (`05-visualizing-results.md`) on that start page.

If/when this PR gets merged, there will be a fair amount of links that will break and need to be patched (e.g., in Learn Metabase articles), but I can handle that downstream work.

Thanks again for your help.

**Comment by d-zh21** [comment]
@jeff-bruemmer 

Is there any way that I could preview my changes locally before I push them?

**Comment by jeff-bruemmer** [comment]
Unfortunately, no; the Metabase website is in a private repo. You can, however, use text editors (emacs, vs code, etc) to preview the markdown as HTML, but that will only approximate how the website will render the file.

File: docs/users-guide/visualizations/row-chart.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+### Row charts
```
Comment (jeff-bruemmer) [inline docs/users-guide/visualizations/row-chart.md:1]: The content on this page page should probably live in the line, bar, and area chart page.

File: docs/users-guide/start.md [hunk 2]
```diff
@@ -22,6 +22,22 @@
 - [Collections](collections.md)
 - [Events and timelines](events-and-timelines.md)
 
+## Visualizing Results
+- [Numbers](visualizations/numbers.md)
```
Comment (jeff-bruemmer) [inline docs/users-guide/start.md:26]:
```suggestion
- [Visualizing results](05-visualizing-results.md)
- [Numbers](visualizations/numbers.md)
```

**Review by jeff-bruemmer — CHANGES_REQUESTED** [review]
A few more changes before we can merge this.

The main viz page (05-visualizing-results.md) shouldn't duplicate the content on the individual pages. That page should function as a landing page for the rest of the viz pages. 

You'll need to:
- delete that duplicate content on the landing page. E.g., the Pivot table section, the trend section, etc.
- keep the introductory info (this page is meant to be a landing page or index page for all the viz sub-pages.
- update the anchor links to page links. E.g., from [Funnel chart](#funnel-chart) to [Funnel chart](visualizations/funnel.md)

If instructions aren't clear, let me know, and I can push up some commits so we can get this over the line. Thanks again for your help, @d-zh21.

File: docs/users-guide/visualizations/row-chart.md (hunk 1 above)
Comment (d-zh21) [reply docs/users-guide/visualizations/row-chart.md]: I integrated the row charts content into the line, bar, and area chart page but the organization of the pages was inconsistent with each other. So I deleted the "Row charts" header and "Histograms" header to make the layout more consistent with the page. Let me know if that's okay

**Comment by jeff-bruemmer** [comment]
@d-zh21 This is good progress. If you can resolve the merge conflicts, and get the tests to pass (you may need to merge master into your branch), we can merge this. I can follow up with another PR to add some metadata to these new files you created.

**Comment by jeff-bruemmer** [comment]
(one year later...)

Hey, @d-zh21. We’ve since overhauled the structure of our docs, which makes this PR a bit stale.

But I didn’t want your work to go to waste here. I started a new PR that incorporates your work here, and then expanded on it with some updates. I mentioned you on the new PR.

**Pull request closed without merging**